metrics/ci_integrations/lib/client/jenkins/mapper/jenkins_build_result_mapper.dart

```diff
@@ -20,6 +21,7 @@ class JenkinsBuildResultMapper {
   const JenkinsBuildResultMapper();
 
   /// Maps the [result] of Jenkins build to the [JenkinsBuildResult].
```

Comment:
Same as above for override methods. Not sure we should document them since they have the same behavior as described in a parent class.